.travis.yml

```diff
@@ -195,6 +199,15 @@ matrix:
           dist: bionic
           stage: Cron tests
           env: SETUP_METHOD='apt'
+               ASTROPY_USE_SYSTEM_ALL=1
+
+        # Temporarily with amd64 and apt, to check.
```

Comment:
Based on the comment here, this job is supposed to be removed before final review?

Reply:
I think this is my only outstanding comment that needs clarification before approval.

Reply:
Yes, and maybe merging this fixes #10561 as well... Let me try to use `s390x` for a single run, and if that works, I'll remove that as well and merge.